Lib/idlelib/pyshell.py

```diff
@@ -1596,6 +1596,9 @@ def main():
     enable_edit = enable_edit or edit_start
     enable_shell = enable_shell or not enable_edit
 
+    # gh-127060: Disable traceback colors
+    os.environ['TERM'] = 'dumb'
```

Comment:
Should not it be set in a subprocess that runs user code?

Reply:
os.environ is inherited, but ok, I modified my PR to only set TERM env var in the subprocess.